### PR TITLE
adding error catches for extra fields

### PR DESCRIPTION
**Pull request by gil-platt**

### CODE REVIEW

File: ckanext/datajson/package2pod.py [hunk 1]
```diff
@@ -3,23 +3,20 @@
...
-import ckan.plugins.toolkit as toolkit
-
 from logging import getLogger
 
-from helpers import *
-from ckan import logic
-from ckan import model
 import ckan.plugins as p
-import ckan
-import urllib
```
Comment (jbrown-xentity) [inline ckanext/datajson/package2pod.py:21]: I don't see these added back in anywhere; are they just unnecessary or were they removed purposefully?

**Review by jbrown-xentity — APPROVED** [review]
Looks pretty good, probably the linting fixes causes it to be hard to parse...

File: ckanext/datajson/package2pod.py (hunk 1 above)
Comment (gil-platt) [reply ckanext/datajson/package2pod.py:21]: they weren't being used
